src/routes/manage.go

```diff
@@ -73,11 +71,9 @@ func (a APIService) editMicrocontroller(c echo.Context) error {
 		fmt.Println(key, value)
 		if key == "description" {
 			fmt.Println("CHANGE DESCRIPTION")
-			a.Cluster.Me.Description = value.(string)
+			//a.Cluster.Me.Description = value.(string)
```

Comment:
The idea is to have a GORM save type thing here?

Reply:
yep 💯

TBH I didn't do much more than make the manage part of the program compile after refactoring. these routes & functions likely need lots of updates